### PR TITLE
Improvements on admission control test to make it blue

### PR DESCRIPTION
**Pull request by fgelcer**
when running this job for 3.2 we saw that it didn't pass.
this makes me investigate more the job, and i found few places that could be improved and fixed.

### CODE REVIEW

File: admission_control_overload_test.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def check_prometheus_metrics(self, start_time, now):
 
         is_admission_control_triggered = False
         for node in node_procs_res:
-            if int(node['values'][0][1]) > 0:
+            if any([values[1] for values in node['values']]) > 0:
```
Comment (bentsi) [inline admission_control_overload_test.py]: `any` returns `True` or `False` and you don't have to put wrap the comprehension with `[`:
```
if any(values[1] for values in node['values'])
```

Comment (fgelcer) [reply admission_control_overload_test.py]: i'm fixing it (and making the test pass in jenkins, as last try it failed) and will update this PR once the test passes.
thank you @bentsi

**Comment by fgelcer** [comment]
@roydahan @bentsi , don't  merge it yet
i didn't push my latest changes to here, yet

**Comment by bentsi** [comment]
@fgelcer please ask for another review and set "ready for review" label when ready

**Comment by bentsi** [comment]
@fgelcer Is this ready?
Please resolve conflicts!

**Comment by fruch** [comment]
@fgelcer 
you have conflict here, and pylint is failing:
```
19:14:45  ************* Module sdcm.cluster
19:14:45  sdcm/cluster.py:1302:20: W0101: Unreachable code (unreachable)
```

**Comment by bentsi** [comment]
@fgelcer please resolve conflicts

File: sdcm/cluster.py [hunk 2]
```diff
@@ -1296,8 +1296,9 @@ def decode_backtrace(self):
             event = None
             obj = None
             try:
-                obj = Setup.DECODING_QUEUE.get(timeout=5)
+                obj = Setup.DECODING_QUEUE.get(timeout=5) if Setup.DECODING_QUEUE else None
                 if obj is None:
+                    break
```
Comment (bentsi) [inline sdcm/cluster.py]: ???? 
break
...
break

File: sdcm/remote.py [hunk 3]
```diff
@@ -609,7 +609,7 @@ def _make_rsync_cmd(self, src, dst, delete_dst, preserve_symlinks):
         """
         ssh_cmd = _make_ssh_command(user=self.user, port=self.port,
                                     hosts_file=self.known_hosts_file,
-                                    key_file=self.key_file,
+                                    key_file=self.key_file, connect_timeout=600,
```
Comment (bentsi) [inline sdcm/remote.py]: need to expose connect_timeout in the calling method

File: sdcm/cluster.py [hunk 4]
```diff
@@ -1296,8 +1296,9 @@ def decode_backtrace(self):
             event = None
             obj = None
             try:
-                obj = Setup.DECODING_QUEUE.get(timeout=5)
+                obj = Setup.DECODING_QUEUE.get(timeout=5) if Setup.DECODING_QUEUE else None
```
Comment (bentsi) [inline sdcm/cluster.py]: irrelevant, this was fixed in  11e580a74c7f83db6877eca411653d9c2b7983c8

File: sct.py [hunk 5]
```diff
@@ -456,6 +456,10 @@ def send_email(test_id=None, email_recipients=None, logdir=None):
     from sdcm.send_email import (GeminiEmailReporter, LongevityEmailReporter,
                                  get_running_instances_for_email_report,
                                  read_email_data_from_file)
+    LOGGER.info(email_recipients)
```
Comment (bentsi) [inline sct.py]: printing variable without a proper message

File: admission_control_overload_test.py [hunk 6]
```diff
@@ -90,15 +101,19 @@ def write_admission_control_load(self):
...
     def test_admission_control(self):
         self.write_admission_control_load()
 
+        # giving some time for the load to calm down
+        time.sleep(10)
         node = self.db_cluster.nodes[0]
         node.stop_scylla(verify_up=False, verify_down=True)
-        node.start_scylla()
+        node.start_scylla(verify_up=True)
+        self.log.info('will give few seconds to everything come back to it\'s place')
```
Comment (bentsi) [inline admission_control_overload_test.py]: In English, when you start a sentence, the first letter should be capital

File: admission_control_overload_test.py [hunk 7]
```diff
@@ -17,12 +17,21 @@ def check_prometheus_metrics(self, start_time, now):
...
         is_admission_control_triggered = False
         for node in node_procs_res:
-            if int(node['values'][0][1]) > 0:
+            if any(values[1] for values in node['values']):
                 self.log.info('Admission control was triggered')
                 is_admission_control_triggered = True
 
         return is_admission_control_triggered
 
+    def kill_load(self):
```
Comment (bentsi) [inline admission_control_overload_test.py]: Why you didn't use `sdcm.cluster.BaseLoaderSet.kill_stress_thread`?

**Pull request closed without merging**